### PR TITLE
Add IVFPQ and RNSG indexes and restructure Milvus Indexes topic

### LINKED ISSUE #280: [Suggestion] Add IVFPQ, RNSG and SPTAG to Milvus Indexes
> Note: This repository is ONLY used to solve issues related to DOCS.
> For other issues, please move to [other repositories](https://github.com/milvus-io/).

**Is there anything that's missing or inappropriate in the docs? Please describe.**
There's no doc about some Milvus indexes such as IVFPQ, RNSG and SPTAG.

**Describe your suggestion**
Add description of these indexes in Milvus Indexes topic.

### PR DESCRIPTION
**Pull request by jielinxu**
This fixes #280 and #277

### CODE REVIEW

File: site/en/reference/milvus_config.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ In the directory `home/$USER/milvus/conf`, open Milvus service configuration fil
 | `gpu_search_threshold` | A Milvus performance tuning parameter. The threshold value must be compared with `nq` to decide if the search computation will be executed on GPUs only.<br/>If `nq` >= `gpu_search_threshold` , the search computation will be executed on GPUs only.<br/>If `nq` < `gpu_search_threshold` , the search computation will be executed on both CPUs and GPUs. | Interger | `1000` |
 </div>
```
Comment (yamasite) [inline site/en/reference/milvus_config.md]: Integer

Comment (jielinxu) [reply site/en/reference/milvus_config.md]: Thx. Already corrected the typo.

File: site/zh-CN/guides/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+- nq：查询的目标向量条目数，在搜索时定义。
+- `nlist`: 聚类时总的分桶数，在创建索引时定义。特征空间被聚类为 `nlist` 个桶。
+- `nprobe`: 查询时需要搜索的分桶数目，在搜索时定义。
+- `gpu_search_threshold`: Milvus 性能调优参数。此参数必须与 nq 比较以确定搜索计算是否只在 GPU 上进行。如果 nq >= `gpu_search_threshold`，则搜索计算只在 GPU 上进行。如果 nq < `gpu_search_threshold`，则搜索计算将在 CPU 和 GPU 上协同进行。
+
+## 索引概览
+
+<div class="table-wrapper" markdown="block">
+
+| 类型                      | Class name | 主要影响参数                                  | 仅支持 CPU 的 Milvus | 支持GPU的Milvus |
```
[the remark this answers is not in the document]
Comment (jielinxu) [reply site/zh-CN/guides/index.md]: Updated the affecting parameters.

File: site/zh-CN/guides/index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## 如何选择索引
+
+若要为您的使用场景选择合适的索引，请参阅 [如何选择索引类型](https://milvus.io/cn/blogs/2019-12-03-select-index.md)。
+
+## 已知问题
+
+#### `IVF_SQ8H`
+
+使用 `IVF_SQ8H` 索引时，如果创建索引和查询同时在 GPU 上进行可能会导致系统崩溃。
```
[the remark this answers is not in the document]
Comment (jielinxu) [reply site/zh-CN/guides/index.md]: Removed the Known issues section.

**Pull request merged**